sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Infrastructure/EventProcessorManager.cs

```diff
@@ -275,7 +275,7 @@ public async Task WaitStabilization()
         ///   Filtering expired ownership is assumed to be responsibility of the caller.
         /// </remarks>
         ///
-        private bool AreOwnershipDistributionsTheSame(IEnumerable<PartitionOwnership> first,
+        internal static bool AreOwnershipDistributionsTheSame(IEnumerable<PartitionOwnership> first,
```

Comment:
This is definitely something we should move to the `Infrastructure` folder I mentioned in another comment. We are going to change some stuff in the `EventProcessorClient` testing in the near future, and I'm not entirely sure if we'll be keeping this method. For the time being, I think we should keep it as a `private` method and just create a copy in the `MockEventProcessorClient`.